fmriprep/workflows/base.py

```diff
@@ -34,15 +34,15 @@ def base_workflow_enumerator(subject_list, task_id, settings, run_uuid):
     if settings['freesurfer']:
         fsdir = pe.Node(BIDSFreeSurferDir(), name='BIDSFreesurfer')
         fsdir.inputs.freesurfer_home = os.getenv('FREESURFER_HOME')
-        fsdir.inputs.derivatives = os.path.join(settings['output_dir'], 'derivatives')
+        fsdir.inputs.derivatives = os.path.join(settings['output_dir'])
 
     generated_list = []
     for subject in subject_list:
         generated_workflow = base_workflow_generator(subject, task_id=task_id,
                                                      settings=settings)
         if generated_workflow:
             generated_workflow.config['execution']['crashdump_dir'] = (
-                os.path.join(settings['output_dir'], 'log', subject, run_uuid)
+                os.path.join(settings['output_dir'], "fmriprep", "sub-" + subject, 'log', subject, run_uuid)
```

Comment:
Does this work correctly when you specify `--participant-label sub-01` (or similar), or do you end up with `sub-sub-01`?